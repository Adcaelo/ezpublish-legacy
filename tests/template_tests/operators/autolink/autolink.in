{raw autolink('Blah blah http://www.example.com blah hello@example.com blah.')}
