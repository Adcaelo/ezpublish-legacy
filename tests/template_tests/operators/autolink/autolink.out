Blah blah <a href="http://www.example.com" title="http://www.example.com">http://www.example.com</a> blah <a href='mailto:hello@example.com'>hello@example.com</a> blah.
